apps/docs/docs/Reference/framework-less/creating-main-file.mdx

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating the main file
+
+Firstly open your code editor and create a new file called `main.mjs` in a folder named "src" of your project. This is where we'll be writing our bot's code.
+
+<Tabs>
+
+<TabItem value="Deno" label="Deno">
+
+```js
+import { App } from 'disploy';
+import { config as loadEnv } from 'std/dotenv/mod.ts';
+import commands from './commands/commands.mjs';
+
+await loadEnv({
+	export: true,
+});
+```
+
+</TabItem>
+
+<TabItem value="Node" label="Node">
+
```

Comment:
ditto but instead explain to install deps with npm, yarn, etc

Reply:
resolved in https://github.com/Disploy/disploy/pull/62/commits/92e7fa9f743a72e97d645e70e7dffc8a3e277527